queries.sql

```diff
@@ -8,3 +8,10 @@ SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true ;
 SELECT * FROM animals WHERE name IS DISTINCT FROM 'Gabumon' ;
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3 ;
+
```

Comment:
You put the correct sql commands in the propper files ✔

* Everything that creates or change the tables structure should be saved in the schema.sql file.
* Everything that populates tables with data should be saved in the data.sql file.
* Everything that is a query that helps to analyze data should be saved in the queries.sql file.